src/pages/settings.tsx

```diff
@@ -8,4 +9,6 @@ const Settings = () => {
   );
 };
 
-export default Settings;
+const protectedSettings = requiresAdminAuth(Settings);
```

Comment:
Instead of doing this for every page, I think it would be better to put it on one component that's always available. Maybe the new Layout component?

Reply:
Yeah, that would be more aligned with DRY principles. I'll get on it!